less/jumbotron.less

```diff
@@ -5,13 +5,17 @@
 
 .jumbotron {
   padding: @jumbotron-padding 0;
-  color: @jumbotron-color;
-  background-color: @jumbotron-bg;
   transition: background-color 0.2s ease-in;
 
-  h1,
-  .h1 {
-    color: @jumbotron-heading-color;
+  &.bg-primary, &.bg-info, .jumbotron-image & {
```

Comment:
I don't think it's necessary to have bg-primary and bg-info here anymore.  Thos classes should change heading colours etc already.